source/github-helpers/index.ts

```diff
@@ -2,7 +2,7 @@ import select from 'select-dom';
 import onetime from 'onetime';
 import elementReady from 'element-ready';
 import compareVersions from 'tiny-version-compare';
-import * as pageDetect from 'github-url-detection/esm/index.js'; // eslint-disable-line import/extensions -- Required for Node tests compatibility
+import * as pageDetect from 'github-url-detection';
```

Comment:
https://github.com/fregante/github-url-detection/pull/37 🎉 hoping this isn't a breaking release for others